src/store/mutations.js

```diff
@@ -4,6 +4,29 @@ import { findDirectory } from '../plugins/helpers';
 import { colorArray, colorChamber, colorHeaterBed/*, temperaturChartSampleLength*/ } from './variables';
 import defaultPrinter from './printer';
 
+let consoleFilterCache = [];
+
+function updateConsoleFilterCache(filters) {
+    consoleFilterCache = [];
+    [...new Set(filters)].forEach(filter => {
+        if (filter && filter.length) {
+            try {
+                const expr = new RegExp(filter);
+                consoleFilterCache.push((t) => expr.test(t));
+            } catch {
+                console.error('Error compiling filter: ' + filter);
+            }
+        }
+    });
+}
+
+function filterConsoleMessage(text) {
```

Comment:
you can move this function to plugins/helpers and only import these into the mutations